assignments/r-10.md

```diff
@@ -6,4 +6,14 @@ language: R
 exercises: ['Higher Order Functions', 'Regular Expressions', 'Debugging', 'Tests']
 ---
 
+
+####Learning Objectives
+
+> Following this assignment students should be able to:
+
+> - write basic and linked functions
```

Comment:
I'm not sure to be honest. I thought I got the phrase from a previous version of the course. It's the subtitle to Functions-4-R.md. But if it wasn't from you... I think we're trying to say 'nested' functions. Though I am also unclear if there is a difference between 'nested' and 'higher order' functions.

I've revised to 'nested functions'.